components/Item/README.md

```diff
@@ -0,0 +1,18 @@
+# Item
+
+Each item is an element from the list of items.
+
+## Item call:
+
+```
+<Item key={id} name={name} />
```

Comment:
Instead of taking name array, take individual fields.

```diff
- <Item key={id} name={name} />
+ <Item
+    key={id}
+    name={name}
+    imageUrl={imageUrl}
+    originalPrice={originalPrice}
+    discountedPrice={discountedPrice}
/>
```

Reply:
Okay!